Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:latest as base
```

Comment:
I recently discovered the potential dangers (lack of backwards compatibility) of selecting the latest image, so I'm thinking it might be a good idea to pin a version!

Reply:
This is always an interesting trade off - using latest certainly means the image can change under you, but pinning too precisely will mean you have to consciously schedule in upgrades. As a middle ground, using a tag like `python:3.11` can mean that you'll pick up patch upgrades automatically but that minor versions will need a conscious bump